ext/mbstring/mbstring.stub.php

```diff
@@ -98,23 +98,23 @@ function mb_ereg(string $pattern, string $string, &$registers = null): int|false
 /** @param array $registers */
 function mb_eregi(string $pattern, string $string, &$registers = null): int|false {}
 
-function mb_ereg_replace(string $pattern, string $replacement, string $string, ?string $option = null): string|false|null {}
+function mb_ereg_replace(string $pattern, string $replacement, string $string, ?string $options = null): string|false|null {}
```

Comment:
The similarity with the pcre extension was already mentioned. In that context I would probably rename `$option` to `$modifiers`. 

However that would also mean the documentation text needs updating. Which... would not be a bad idea anyways. It speaks about "options" and then lists all possible "options" which also include "modes". So options can also contain a mode. Confusing.

https://www.php.net/manual/en/function.mb-regex-set-options.php

Maybe it should speak about "behavior modifiers" and "syntax modifiers" in stead.

Reply:
Right, modifiers is a possible alternative term. However, the referenced function is called mb_regex_set_options(), so that seems like the preferred term for mb_regex. (PCRE also never exposes "modifiers" in parameters, as they are part of the regex -- as such, there is no naming consistency to upkeep in that respect at least.)